### PR TITLE
Add basic level logic based on enemies in db

### PR DESCRIPTION
**Pull request by mfmendiola**
**What**

- Change ship shape to cross
- Add logic to load levels with enemies from DB
- Clean up semicolons
- Destroy enemies on collision with your ship
- Cap one enemy on screen at at tme
- Kill enemy  when offscreen

### CODE REVIEW

File: js/engine.js [hunk 1]
```diff
@@ -61,56 +61,52 @@ function init() {
...
-  }
 
-  let loadButton = document.getElementById('loadButton');
+  let loadButton = document.getElementById('loadButton')
 
   loadButton.addEventListener('click', async function () {
-    const shape = await newShape.loadShapeObject();
-    shapesOnScreen[shape.name] = shape;
-  });
+    const shape = await newShape.loadShapeObject()
```
Comment (michaelwschultz) [inline js/engine.js]: Nit - wrong indention

File: js/classes.js [hunk 2]
```diff
@@ -253,21 +271,43 @@ class UserShip extends Ship {
 
 class EnemyShip extends Ship {
   constructor(shape, name, health) {
-    super(shape, name, health)
+    super(shape, name, null, health)
```
Comment (michaelwschultz) [inline js/classes.js]: Why the addition of `null` here?

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+.git
```
Comment (michaelwschultz) [inline .gitignore]: You want to ignore the git folder itself?

Comment (mfmendiola) [reply .gitignore]: Let  me  revert this. Was futzing  with stuff toget t hings  to work

File: js/engine.js [hunk 4]
```diff
@@ -171,23 +171,31 @@ function render() {
...
+
     for (let b = 0; b < myShip.bullets.length; b++) {
       turnOn(myShip.bullets[b].location, "bg-red")
     }
 
     // show enemies on screen
-    for (let j = 0; j < enemies.length; j++) {
-      for (let i = 0; i < enemies[j].shape.shapeType.length; i++) {
+
+      for (let i = 0; i < enemies[0].shape.shapeType.length; i++) {
```
Comment (michaelwschultz) [inline js/engine.js]: Nit - indention

File: js/classes.js (hunk 2 above)
Comment (mfmendiola) [reply js/classes.js]: Changing to accept color. It's cuz `EnemyShip` doesn't aceppt a color but Ship does  (https://github.com/michaelwschultz/space-shooter-led/pull/12/files#diff-c98a8a6f8fc25f2f4c08ff43eb922499R107)

**Pull request merged**